test/test_in_toto_verify.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+
+"""
+<Program Name>
+  test_in_toto_verify.py
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Jan 9, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Test in_toto_verify command line tool.
+
+"""
+
+import os
+import sys
+import unittest
+import logging
+import argparse
+import shutil
+import tempfile
+from mock import patch
+
+from in_toto.models.link import Link
+from in_toto.models.layout import Layout
+from in_toto.in_toto_verify import main as in_toto_verify_main
+from in_toto.in_toto_verify import in_toto_verify
+from in_toto import log
+from in_toto import exceptions
+from in_toto.util import import_rsa_key_from_file
+
+
+# Suppress all the user feedback that we print using a base logger
+logging.getLogger().setLevel(logging.CRITICAL)
+
+class TestInTotoVerifyTool(unittest.TestCase):
+  """
+  Tests
+    - in_toto_verify's main() - requires sys.argv patching;
+    - in_toto_verify - calls verifylib.in_toto_verify and error logs/exits
+      in case of a raised Exception.
+
+  Uses in-toto demo supply chain link metadata files and basic layout for
+  verification:
+
+  Copies the basic layout for different test scenarios:
+    - single-signed layout
+    - double-signed layout
```

Comment:
This is fine (it's not specified yet), and I don't see it as a blocker. I think it'd be clearer if we called this (multiple-signatures-on-layout) and then specify it's 2 to convey the fact that it supports multiple signature, not just one case or the other.